arbitrum-docs/run-arbitrum-node/more-types/02-run-validator-node.mdx

```diff
@@ -1,76 +1,150 @@
+
+| Strategy           | Description                                                                                                                                                                                                                                                                                                                                                   | Gas usage                                                                                               |
+| ------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------- |
+| **`Defensive`**    | If the local state disagrees with the on-chain assertion, this validator will post a stake and create a challenge                                                                                                                                                                                                                                             | Only acts if a bad assertions is found                                                                  |
+| **`StakeLatest`**  | This validator will stay staked on the latest assertion found, and will challenge any bad assertions that it finds (this strategy is only available in pre-BoLD chains)                                                                                                                                                                                       | Gas used every time a new assertion is created                                                          |
+| **`ResolveNodes`** | This validator will stay staked on the latest assertion found, resolve any unconfirmed assertions, and it will challenge any bad assertions that it finds                                                                                                                                                                                                     | Gas used every time a new assertion is created, and to resolve unconfirmed assertions                   |
+| **`MakeNodes`**    | This validator continuously creates new assertions, resolves any unconfirmed assertions, and challenges bad assertions found. Note that if there is more than one `MakeNodes` validator running, they might all try to create a new assertion at same time. In that case, only one will be successful, while the others will have their transactions reverted | Gas used to create new assertions, move the stake to the latest one, and resolve unconfirmed assertions |
```

Comment:
```suggestion
| ------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------- |
| **`Defensive`**    | If the local state disagrees with the on-chain assertion, this validator will post a stake and create a challenge                                                                                                                                                                                                                                             | Only acts if a bad assertions is found                                                                  |
| **`StakeLatest`**  | This validator will stay staked on the latest assertion found and will challenge any bad assertions that it finds (this strategy is only available in pre-BoLD chains)                                                                                                                                                                                       | Gas used every time a new assertion is created                                                          |
| **`ResolveNodes`** | This validator will stay staked on the latest assertion found, resolve any unconfirmed assertions, and it will challenge any bad assertions that it finds                                                                                                                                                                                                     | Gas used every time a new assertion is created and to resolve unconfirmed assertions                   |
| **`MakeNodes`**    | This validator continuously creates new assertions, resolves any unconfirmed assertions, and challenges bad assertions found. Note that if there is more than one `MakeNodes` validator running, they might all try to create a new assertion simultaneously. In that case, only one will be successful, while the others will have their transactions reverted | Gas used to create new assertions, move the stake to the latest one, and resolve unconfirmed assertions |
```